<!DOCTYPE html>
<html lang="en-us">
  <head>
  <link href="http://gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="content-type" content="text/html; charset=utf-8">

  <!-- Enable responsiveness on mobile devices-->
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">

  <title>
    
      Workflows &middot; The Neuro Bioinformatics Core
    
  </title>

  <!-- CSS -->
 	<!-- Bootstrap core CSS -->
  <link rel="stylesheet" href="/public/css/poole.css">
   <link rel="stylesheet" href="/public/css/lanyon.css">

        <link href="/css/blog.css" rel="stylesheet">
	<link rel="stylesheet" href="/css/pygments.css">
  <!-- Icons -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/public/apple-touch-icon-144-precomposed.png">
  <link rel="shortcut icon" href="/public/favicon.ico">

  <!-- RSS -->
  <link rel="alternate" type="application/rss+xml" title="RSS" href="/atom.xml">
</head>


  <body>

    <!-- Target for toggling the sidebar `.sidebar-checkbox` is for regular
     styles, `#sidebar-checkbox` for behavior. -->
<input type="checkbox" class="sidebar-checkbox" id="sidebar-checkbox">

<!-- Toggleable sidebar -->
<div class="sidebar" id="sidebar">
  <div class="sidebar-item">
    <p></p>
  </div>
<div class="contact-buttons" style="line-height:160%;margin-left:30px;margin-top:10px">
<p>
<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
<link rel="stylesheet" href="//neurobioinfo.github.io/css/academicons.css">
  <a href="mailto:neurobioinfo@mcgill.ca" target="_blank" style="color:#855f65;"><i class="fa fa-envelope" style="font-size:1em"></i> &nbsp </a>
<a href="https://github.com/neurobioinfo" target="_blank" style="color:#855f65;"><i class="fa fa-github" aria-hidden="true"></i> &nbsp </a>
<a href="https://www.researchgate.net/profile/Saeid_Amiri6" target="_blank" style="color:#855f65;"><i class="ai ai-researchgate ai-3x" style="font-size:1em"></i> &nbsp <br></a> 
<a href="https://scholar.google.com/citations?user=iroPmuAAAAAJ&hl=en" target="_blank" style="color:#855f65;"><i class="ai ai-google-scholar-square ai-3x" style="font-size:1em"></i> &nbsp </a>
</p>
</div>
     
  <nav class="sidebar-nav">
    <a class="sidebar-nav-item" href="/">Home</a>

    

    
    
        <!-- <a class="sidebar-nav-item" href="https://neurobioinfo.github.io/">The NeuroBioinfo Home</a> -->
        <a class="sidebar-nav-item" href="https://neurobioinfo.github.io/team">Team</a>
        <!-- <a class="sidebar-nav-item" href="https://neurobioinfo.github.io/wf/">Workflows</a> -->
        <a class="sidebar-nav-item" href="https://neurobioinfo.github.io/blog">Workflows</a>
        <a class="sidebar-nav-item" href="https://github.com/neurobioinfo">GitHub</a>
        <!-- <a class="sidebar-nav-item" href="https://neurobioinfo.github.io/software/">Software</a> -->
        <a class="sidebar-nav-item" href="https://neurobioinfo.github.io/faq_def">FAQs & Definitions</a>
        <a class="sidebar-nav-item" href="https://neurobioinfo.github.io/resources/">Useful resources</a>
        <a class="sidebar-nav-item" href="https://neurobioinfo.github.io/papers/">Publications</a>
        <!-- <a class="sidebar-nav-item" href="https://neurobioinfo.github.io/blog">Blog</a> 
        <a class="sidebar-nav-item" href="https://neurobioinfo.github.io/papers/">Technical report</a>--> 
        <!-- <a class="sidebar-nav-item" href="https://neurobioinfo.github.io/tutorials/">Tutorials</a> -->
        <!-- <a class="sidebar-nav-item" href="https://neurobioinfo.github.io/Misc">Misc</a> -->
        <!-- <a class="sidebar-nav-item" href="https://neurobioinfo.github.io/tags">Tags</a> -->

  </nav>

  <div class="sidebar-item">
  </div>
</div>


    <!-- Wrap is the content to shift when toggling the sidebar. We wrap the
         content to avoid any CSS collisions with our real content. -->
    <div class="wrap">
      <div class="masthead">
        <div class="container">
          <label for="sidebar-checkbox" class="sidebar-toggle"></label>

          <h3 class="masthead-title">
            <a href="/" title="Home">The Neuro Bioinformatics Core</a>
            <small></small>
          </h3>
        </div>
      </div>

      <div class="container content">
        <head>
  <link href="http://gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="content-type" content="text/html; charset=utf-8">

  <!-- Enable responsiveness on mobile devices-->
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">

  <title>
    
      Workflows &middot; The Neuro Bioinformatics Core
    
  </title>

  <!-- CSS -->
 	<!-- Bootstrap core CSS -->
  <link rel="stylesheet" href="/public/css/poole.css">
   <link rel="stylesheet" href="/public/css/lanyon.css">

        <link href="/css/blog.css" rel="stylesheet">
	<link rel="stylesheet" href="/css/pygments.css">
  <!-- Icons -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/public/apple-touch-icon-144-precomposed.png">
  <link rel="shortcut icon" href="/public/favicon.ico">

  <!-- RSS -->
  <link rel="alternate" type="application/rss+xml" title="RSS" href="/atom.xml">
</head>


<div class="post">
  

  <article class="post-content">
    <p>#Workflows</p>

<p>On these pages, we describe the workflows we are currently offering as a service to our Neuro community. As described here, they are generic and we do adapt all of these on a case-by-case for each of our specific tasks.</p>

<p>For RNA and DNA sequence analysis, we will often refer to secondary or tertiary analyses in our discussions with you.  Here is what we are referring to, in part taken from Illumina documentation [1], and from Pereira and colleagues [2].</p>

<h2 id="primary-analysis">Primary Analysis</h2>

<p>The primary data analysis consists of the detection and analysis of raw data, targeting the generation of legible sequencing reads (base calling) and scoring base quality. The outputs from this primary analysis are FASTQ ﬁles and are often from the genome center that sequenced your samples, this is usually where we start, doing QC on your samples [2]. It is your responsibility to understand how data is generated.</p>

<h2 id="secondary-analysis">Secondary Analysis</h2>

<p>A secondary analysis will involve a per-sample alignment (mapping to a reference genome, and in 2021/2022, this is usually to the human reference hg38 (also known as GRCh38.p13 [3]) and then do a variant calling. This is typically the generation of a VCF file. This will include calling the various types of variation one can see in a genome: single nucleotide polymorphisms (SNP), small insertion/deletions (INDELS), less than 50 bp, larger structural INDELS (&gt; 50bp), translocations/large deletions, Copy Number Variation (CNV) changes, and larger chromosomal aberrations like trisomies and the likes.</p>

<h2 id="tertiary-analysis">Tertiary Analysis</h2>

<p>A tertiary analysis is everything that comes afterwards, starting with the joint-calling analysis step that allows better variant identification by leveraging population-wide information from cohorts [4]; annotations, filtering, some interpretation (this will be discussed with our team when planning your project). This is actually where a lot of the art of the trade will come into play, and where there will be more variation in the types of analysis we will perform, specific to your project and will inform the analysis we do to inform the functional impact of the variants you have identified in your analysis.</p>

<p>All of this will become clear after you have spoken to us, and a plan is set for the analysis we will be doing for you.</p>

<h2 id="current-workflows">Current workflows</h2>

<p>This is a list of the current workflows we run in the Neuro Bioinformatics Core Facility. If you do not see the workflow you need, or if you want to use one of these or modify them, please do not hesitate to <a href="mailto:neurobioinfo@mcgill.ca">contact us at neurobioinfo@mcgill.ca</a>.</p>

<ul>
  <li><a href="/wf/rna.md">RNA-Seq Analysis</a></li>
  <li><a href="/wf/rna_dea.md">RNA-Seq and Differential Expression Analysis</a></li>
  <li><a href="/wf/mirna.md">miRNA-Seq Analysis</a></li>
  <li><a href="/wf/scrna.md">scRNA-Seq Analysis</a></li>
  <li><a href="/wf/wga.md">Whole-Genome/Exome Mapping and Variant Calling Analysis</a></li>
  <li><a href="/wf/segregation.md">Segregation Analysis</a></li>
  <li><a href="/wf/data_management.md">Data Management</a></li>
</ul>

<h2 id="bibliography">Bibliography</h2>

<ol>
  <li>Illumina documentation <a href="https://www.illumina.com/informatics/sequencing-data-analysis.html">Software to help you focus on your research</a></li>
  <li>Bioinformatics and Computational Tools for Next-Generation Sequencing Analysis in Clinical Genetics. Pereira R, Oliveira J, Sousa M. J Clin Med. 2020 Jan 3;9(1):132.  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/31947757/">31947757</a> </li>
  <li>Genome Reference Consortium for the most recent human genome reference: <a href="https://www.ncbi.nlm.nih.gov/grc/human">GRCh38.p13</a></li>
  <li>From the Broad’s GATK documentation: <a href="https://gatk.broadinstitute.org/hc/en-us/articles/360035890431-The-logic-of-joint-calling-for-germline-short-variants">The logic of joint calling for germline short variants</a></li>
</ol>


  </article>

</div>

      </div>
    </div>
   <script id="dsq-count-scr" src="//neurobioinfo-github-io.disqus.com/count.js" async></script>
  </body>
</html>
